src/routing/pages/probe/Table.svelte

```diff
@@ -5,21 +5,27 @@
   import ProbeTableView from '../../../components/table/ProbeTableView.svelte';
   import ProbeTitle from '../../../components/regions/ProbeTitle.svelte';
   import { store, probe } from '../../../state/store';
+  import { isSelectedProcessValid } from '../../../utils/probe-utils';
+  import DataError from '../../../components/errors/DataError.svelte';
 </script>
 
 <Probe let:data let:probeType>
   <div class="graphic-body__content graphic-body__content--no-padding">
     <div in:fade>
-      <!-- this conditional is a stopgap until we fix https://github.com/mozilla/glam/issues/206 -->
-      {#if $probe.type}
-        <ProbeTableView
-          data={data.data}
-          probeType={probeType}
-          colorMap={data.bucketColorMap}
-          visibleBuckets={[...$store.activeBuckets]}
-          aggregationLevel={$store.aggregationLevel}>
-            <ProbeTitle />
-        </ProbeTableView>
+      {#if isSelectedProcessValid($store.recordedInProcesses, $store.process)}
+        <!-- this conditional is a stopgap until we fix https://github.com/mozilla/glam/issues/206 -->
+        {#if $probe.type}
+          <ProbeTableView
+            data={data.data}
+            probeType={probeType}
+            colorMap={data.bucketColorMap}
+            visibleBuckets={[...$store.activeBuckets]}
+            aggregationLevel={$store.aggregationLevel}>
+              <ProbeTitle />
+          </ProbeTableView>
+        {/if}
+      {:else}
+        <DataError reason={`This probe does not record in the ${$store.process} process.`} />
       {/if}
```

Comment:
If I'm not mistaken, it looks like lines 15, 27, 28, and 29 can be moved to *Probe.svelte* so that we're not repeating that logic in both *Explore.svelte* and *Table.svelte*.